app/jobs/importing/eto_demographics_job.rb

```diff
@@ -15,5 +15,6 @@ def perform(client_ids:)
     def max_attempts
       1
     end
+
```

Comment:
Shouldn't RuboCop have caught this? :-)

Reply:
Only if the change came in on a commit where the action was working.  (Which I'm guessing it didn't.)